lib/libraries_map.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Map library codes to labels exported from Folio
+class LibrariesMap
+  def self.for(key)
+    return if key == 'SUL' # do not index building_facet on electronic resources
+
+    Folio::Types.libraries.values.find { |library| library.fetch('code') == key }.fetch('name')
```

Comment:
This doesn't work for Lane's material:

```
./lib/libraries_map.rb:8:in `for': undefined method `fetch' for nil:NilClass (NoMethodError)

    Folio::Types.libraries.values.find { |library| library.fetch('code') == key }.fetch('name')
                                                                                 ^^^^^^
```

It looks like we're passing in the Symphony library codes (which makes sense... for parity), but the Symphony + FOLIO codes for Lane are different (`LANE` vs `LANE-MED`).

We should account for that difference here, and also add a reasonable fallback if we don't get a name (probably nil so it is ignored?). This is just populating a facet, so it's not the end of the world if we don't get a match for whatever reason.